<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pokedex</class>
 <widget class="QMainWindow" name="Pokedex">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1150</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pokedex</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1150</width>
     <height>750</height>
    </size>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1151</width>
      <height>741</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>200</horstretch>
           <verstretch>200</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tab">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>200</horstretch>
            <verstretch>200</verstretch>
           </sizepolicy>
          </property>
          <attribute name="title">
           <string>Pokedex</string>
          </attribute>
          <widget class="QWidget" name="horizontalLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1151</width>
             <height>711</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3"/>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QTableView" name="tableView">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>600</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string extracomment="*It's possible to filter the pokedex based on pokemon types">Types :</string>
                </property>
                <property name="textFormat">
                 <enum>Qt::RichText</enum>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="fighting">
                <property name="text">
                 <string>FIGHTING</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="dark">
                <property name="text">
                 <string>DARK</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="ghost">
                <property name="text">
                 <string>GHOST</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="flying">
                <property name="text">
                 <string>FLYING</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="ground">
                <property name="text">
                 <string>GROUND</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="fairy">
                <property name="text">
                 <string>FAIRY</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="steel">
                <property name="text">
                 <string>STEEL</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="rock">
                <property name="text">
                 <string>ROCK</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="water">
                <property name="text">
                 <string>WATER</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="bug">
                <property name="text">
                 <string>BUG</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="fire">
                <property name="text">
                 <string>FIRE</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="psychic">
                <property name="text">
                 <string>PSYCHIC</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="electric">
                <property name="text">
                 <string>ELECTRIC</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="normal">
                <property name="text">
                 <string>NORMAL</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="poison">
                <property name="text">
                 <string>POISON</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="dragon">
                <property name="text">
                 <string>DRAGON</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="ice">
                <property name="text">
                 <string>ICE</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="grass">
                <property name="text">
                 <string>GRASS</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <attribute name="title">
           <string>Comparison Tool</string>
          </attribute>
          <widget class="QWidget" name="verticalLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1141</width>
             <height>701</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3"/>
          </widget>
          <widget class="QWidget" name="horizontalLayoutWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1276</width>
             <height>750</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>SELECT POKEMON 1:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListWidget" name="listWidget">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>400</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="pokemon1label">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>300</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>300</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <item>
               <widget class="QWidget" name="radarChart" native="true">
                <property name="minimumSize">
                 <size>
                  <width>600</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>600</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>SELECT POKEMON 2:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListWidget" name="listWidget_2">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>400</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="pokemon2label">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>300</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>300</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
